doc/source/user_guide/numpyro_refitting_xr_lik.ipynb

```diff
@@ -13,7 +13,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Below there is one example of `SamplingWrapper` usage for NumPyro."
+    "Below there is an example of `SamplingWrapper` usage for [NumPyro] (https://pyro.ai/numpyro/)."
```

Comment:
remove the space between [](). I added it because the markdown was rendering it to link.